buyer-api/src/operations/getData.js

```diff
@@ -0,0 +1,32 @@
+import { parseAsync } from 'json2csv';
+import { getRawOrderData } from '../utils/wibson-lib/s3';
+
+const NOT_FOUND = {
+  code: 'getData.not_found',
+  message: "The data couldn't get",
```

Comment:
```suggestion
  message: "The data couldn't be found",
```

Reply:
I think the message that goes here should be other than `The data couldn't be found`

Reply:
Another thing I want to insist on, it's better to describe the error here and decide what status code the service needs to respond in the express handler.